_sass/blocks/_revenues_subpage_steps.scss

```diff
@@ -80,8 +93,10 @@ $onshore-renewables-secondary-color: $blue;
   }
 
   @include respond-to(medium-up) {
-    @include span-columns(7);
-    @include shift(2);
+    @include direction-context(right-to-left) {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets